packages/neon_framework/packages/account_repository/test/account_repository_test.dart

```diff
@@ -0,0 +1,564 @@
+              b.poll
+                ..token = 'token'
+                ..endpoint = 'token';
+            }),
+            null,
+          );
+        });
+        await repository.loginFlowInit(Uri());
+        when(() => clientFlowLoginV2.poll($body: any(named: r'$body'))).thenThrow(http.ClientException(''));
+
+        await expectLater(
+          repository.loginFlowPoll(),
+          throwsA(isA<PollLoginFailure>().having((e) => e.error, 'error', isA<http.ClientException>())),
+        );
+      });
+    });
+
+    group('getAccount', () {
+      test('retrieves account id from server', () async {
+        when(() => users.getCurrentUser()).thenAnswer(
+          (_) async => DynamiteResponse(
+            200,
+            provisioning_api.UsersGetCurrentUserResponseApplicationJson.fromJson({
+              'ocs': {
+                'meta': {'status': 'ok', 'statuscode': 200, 'message': 'OK'},
+                'data': {
+                  'additional_mail': [],
+                  'address': '',
+                  'backend': 'Database',
+                  'backendCapabilities': {'setDisplayName': true, 'setPassword': true},
+                  'biography': '',
+                  'display-name': 'admin',
+                  'displayname': 'admin',
+                  'fediverse': '',
+                  'groups': ['admin'],
+                  'headline': '',
+                  'id': 'admin',
+                  'language': 'en',
+                  'lastLogin': 1723124410000,
+                  'locale': '',
+                  'manager': '',
+                  'organisation': '',
+                  'phone': '',
+                  'profile_enabled': '1',
+                  'quota': {
+                    'free': 1629916880896,
+                    'relative': 0,
+                    'total': 1629956170522,
+                    'used': 39289626,
+                    'quota': -3,
+                  },
+                  'role': '',
+                  'subadmin': [],
+                  'twitter': '',
+                  'website': '',
+                  'additional_mailScope': [],
+                  'addressScope': 'v2-local',
+                  'avatarScope': 'v2-federated',
+                  'biographyScope': 'v2-local',
+                  'displaynameScope': 'v2-federated',
+                  'email': 'admin@example.com',
+                  'emailScope': 'v2-federated',
+                  'enabled': true,
+                  'fediverseScope': 'v2-local',
+                  'headlineScope': 'v2-local',
+                  'organisationScope': 'v2-local',
+                  'phoneScope': 'v2-local',
+                  'profile_enabledScope': 'v2-local',
+                  'roleScope': 'v2-local',
+                  'storageLocation': '/usr/src/nextcloud/data/admin',
+                  'twitterScope': 'v2-local',
+                  'websiteScope': 'v2-local',
+                },
+              },
+            }),
```

Comment:
Any reason to not mock it?

Reply:
It would be a lot of mocks (OCS, OCSdata, UserDetails, ...)
Should I mock it nevertheless? Now that I think about it 3 Mocks would probably be cleaner than these 50 lines.

Reply:
Yes that would better